### DISCUSSION TITLE
Improve the Contributors Section in Release Tiddler

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6783] [Ideas]
Following to discussion here on Forum: https://talk.tiddlywiki.org/t/tw-5-2-3-is-getting-ready-to-be-released/3867/12?u=mohammad and few solutions I like to propose a minor improvement to Release tiddler

This solution has

1. A small modification to contributor macro by adding a dive
2. Adding few css to use flex section with 80x80px children
3. Use a $list widget to sort and display contributors
4. The $list uses sort, so no matter how to add contributor names
5. It is possible to use a tiddler or a field to keep the list of contributors, here I used a variable...

To give a try download [Contributors - ii.zip](https://github.com/Jermolene/TiddlyWiki5/files/9112733/Contributors.-.ii.zip) and unzip and drag and drop to https://tiddlywiki.com/prerelease/

Tested on a separate tiddler on https://tiddlywiki.com/prerelease/

![image](https://user-images.githubusercontent.com/830394/179010013-6b9e3ee5-8dcb-4d32-b284-8ba02dced84b.png)

@Jermolene - if you like the solution let me to submit a PR. The solution is customizable and adds few bytes to Release tiddler

**Comment by pmario** [comment]
I think this could have been an issue too

**Comment by kookma** [comment]
@pmario 
an issue has been created

https://github.com/Jermolene/TiddlyWiki5/issues/6784
